### ISSUE TITLE
[Community style] Some code blocks illegible due to insufficient color contrast

### ISSUE THREAD

**Issue by btat** [rancher/rancher-product-docs#1199]
This only affects some, but not all, code blocks.

Example(s):
- The HTML tags in this [HTML example](https://rancher.github.io/rancher-product-docs/getting-started/quick-start-guides/deploy-workloads/nodeports#_attention_cloud_hosted_sandboxes)
    - The [Product version](https://documentation.suse.com/cloudnative/rancher-manager/latest/en/installation-and-upgrade/quick-start/deploy-workloads/nodeports.html#_attention_cloud_hosted_sandboxes) of the same page
- The `-` characters in this [YAML example](https://rancher.github.io/rancher-product-docs/reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters#_impersonation_troubleshooting) (to a lesser extent)
  - The [Product version](https://documentation.suse.com/cloudnative/rancher-manager/latest/en/about-rancher/architecture/communicating-with-downstream-clusters.html#_impersonation_troubleshooting) of the same page

**Comment by btat** [comment]
Addressed by #1233.
